src/Apps/Artist/Components/ArtistConsignButton.tsx

```diff
@@ -0,0 +1,128 @@
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+import { ArtistConsignButton_artist } from "__generated__/ArtistConsignButton_artist.graphql"
+import { RouterLink } from "Artsy/Router/RouterLink"
+import { Media } from "Utils/Responsive"
+import { getConsignmentData } from "../Routes/Consign/Utils/getConsignmentData"
+
+import {
+  BorderBox,
+  Box,
+  Button,
+  color,
+  Flex,
+  Image,
+  Sans,
+} from "@artsy/palette"
+
+export interface ArtistConsignButtonProps {
+  artist: ArtistConsignButton_artist
+}
+
+export const ArtistConsignButton: React.FC<ArtistConsignButtonProps> = props => {
+  return (
+    <>
+      <Media at="xs">
+        <ArtistConsignButtonSmall {...props} />
+      </Media>
+      <Media greaterThan="xs">
+        <ArtistConsignButtonLarge {...props} />
+      </Media>
+    </>
+  )
+}
+
+export const ArtistConsignButtonLarge: React.FC<ArtistConsignButtonProps> = props => {
+  const { isTop20, imageURL, headline, consignLink } = getData(props)
+
+  return (
+    <BorderBox p={1} width="100%">
+      <Flex alignItems="center" width="100%" justifyContent="space-between">
+        <Flex>
+          {isTop20 && <Image src={imageURL} width={50} height={50} />}
+          <Flex flexDirection="column" justifyContent="center" pl={1}>
+            <Sans size="3t" weight="medium">
+              {headline}
+            </Sans>
+            <Box position="relative">
+              <Sans size="3t" color={color("black60")}>
```

Comment:
Something I've always wondered - some reaction components use `color={color("black60")}`, and some use `color="black60"`. Is there a reason to favor one over the other?

Reply:
I guess in this case we don't _need_ `color` because this component will only ever live inside of our app, which is contained by boot, but color makes it failproof should it be moved, etc in the future, while also giving us intellisense.